pkg/datasource.go

```diff
@@ -817,13 +817,13 @@ func (o *OCIDatasource) processLogMetrics(ctx context.Context, searchLogsReq Gra
 
 									// Check whether the key contains one of the aggregation functions
 									if key == "count" {
-										metricFieldName = "count"
+										metricFieldName = ""
```

Comment:
Generally these changes work but there are a couple of special / corner cases where simply clearing the `metricFieldName` variable doesn't lead to the ideal result as the legend shows 'Value'. The use cases in question are:

- Simple query with just a `count` operation:  `search "<compartment OCID>" | count`
![Screen Shot 2022-08-03 at 6 56 26 PM](https://user-images.githubusercontent.com/37004987/182746505-7c210e54-c317-48a9-9af8-1825af6c908f.png)

- Similar query with just a `summarize count()` operation:  `search "<compartment OCID>" | summarize count()`
![Screen Shot 2022-08-03 at 6 56 17 PM](https://user-images.githubusercontent.com/37004987/182746522-fa005a18-d1be-42ca-84ec-dc36bc6b4cc6.png)

- Another variation of the query with a field specified:  `search "<compartment OCID>" | summarize count(<Field Name>)`
![Screen Shot 2022-08-03 at 6 34 08 PM](https://user-images.githubusercontent.com/37004987/182746310-9011c9af-0a94-4ceb-8faf-d065c058440f.png)

This is probably okay for now but ultimately the plugin should generate a more appropriate legend value for these cases. I'll accept this PR if a bug issue is written to identify the follow on work that needs to be done.

Reply:
Previously, without the changes of this PR, the output was like this for a simple summarize count: 
![image](https://user-images.githubusercontent.com/106824979/182788851-50b919fe-4a7d-4b94-9247-ad347b8361c3.png)
which also is not ideal in my opinion. I think this PR will introduce a better label generation in many graphs compared to the main branch, and I do agree we need a follow-up to handle these three special  cases.